projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -399,6 +399,7 @@
 				border-radius: var(--jetpack--contact-form--button-outline--border-radius);
 				padding: var(--jetpack--contact-form--button-outline--padding);
 				line-height: var(--jetpack--contact-form--button-outline--line-height);
+				align-items: center;
```

Comment:
Align items in the center in _Button_ mode so the padding look similar at the top and bottom.